Firestore/core/test/firebase/firestore/auth/firebase_credentials_provider_test.mm

```diff
@@ -25,15 +31,41 @@ namespace auth {
 // Set a .plist file here to enable the test-case.
 static const char* kPlist = "";
```

Comment:
This seems like an integration test for actual auth, no?

If so, we probably need a better mechanism for supporting this kind of usage. The values required for testing in the nightly/qa testing environments are passed via environment variables. I wonder if we should do something similar here?

For now it's probably sufficient to add a TODO to wire this up into CI.

Reply:
ack